### DISCUSSION TITLE
[Support]: Audio detection - How to map between decibels reported in HomeAssistant and the 0-1000 min_volume in Frigate?

### DISCUSSION THREAD

**Discussion by rsteckler** [blakeblackshear/frigate#13441] [General Support]
### Describe the problem you are having

I'd like to tune my min_volumes for audio detection in Frigate.  Frigate has a min_volume setting of 0-1000, but Home Assistant reports the current volume in decibels.  
Is it possible to map between the two so I can monitor realtime in HA and make changes in Frigate?

### Version

latest

### Frigate config file

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Relevant go2rtc log output

```shell
NA
```

### FFprobe output from your camera

```shell
NA
```

### Operating system

Other Linux

### Install method

Docker Compose

### Object Detector

Coral

### Network connection

Wired

### Camera make and model

various dahua

**Comment by NickM-27** [comment]
I'd suggest using MQTT explorer and watching the RMS value directly
